Source/Base/Base/Math/Interpolation.h

```diff
@@ -0,0 +1,403 @@
+#include "Math.h"
+
+namespace Spline
+{
+    class Interpolation
```

Comment:
Prefer the use of a namespace if we are dealing with strictly static functions in cases like these. (This goes for the other classes/structs too that only hold static functions in this file)